docs/commands.md

```
@@ -57,7 +58,28 @@ CLI サブコマンドとチャットスラッシュコマンドの完全仕様
 | 終了コード | 成功時 `0`、AlreadyRunning 時 `1`（stderr にメッセージ出力） |
 | 動作 | 新規メッセージ数が閾値（≤ 4）以下の場合はスキップして終了 |
 
-### 1.5 `egopulse gateway status`
+### 1.5 `egopulse events extract`
+
+過去セッションの会話履歴からエピソードイベントを抽出する。イベントテーブルが不完全な場合のバックフィルに使用する。
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)